components/Button/button.css

```diff
@@ -0,0 +1,51 @@
+@import url("../../Stylesheet/variable.css");
+
+.btn {
+  position: relative;
+  font-family: inherit;
+  box-sizing: border-box;
+  cursor: pointer;
+  font-weight: var(--text-bold);
+  color: var(--color-white);
+  outline: auto;
+  border-radius: 0.5rem;
+  box-shadow: 3px 3px 0px rgb(119 136 153);
```

Comment:
Add ` , ` to separate the values inside `rgb ()`

Reply:
Thanks for pointing this out have fixed it.